app/controllers/v0/benefits/document_uploads_controller.rb

```diff
@@ -0,0 +1,155 @@
+            schema do
+              property :id, type: :string, example: 'b62e1379-e703-4aa7-a9dd-c74e66710f46', description: 'Identifier for subsequent getStatus requests'
+            end
+          end
+
+          response 400, description: 'Bad Request' do
+            key :'$ref', :DocumentUploadErrors
+          end
+
+          response 401, description: 'Unauthorized Request'
+          response 403, description: 'Bad API Token'
+        end
+      end
+
+      swagger_schema :DocumentMetadata do
+        key :required, %i[veteranFirstName veteranLastName fileNumber
+                          receiveDate zipCode source uuid docType
+                          numberPages hashV numberAttachments]
+
+        property :veteranFirstName, type: :string, example: 'Jane'
+        property :veteranLastName, type: :string, example: 'Doe'
+        property :fileNumber, type: :string, example: '999887777', description: '8 or 9 digit number'
+        property :receiveDate, type: :string, example: '2001-01-01 12:00:00', description: 'Date/time document was received in US Central Time'
+        property :zipCode, type: :string, example: '20571'
+        property :source, type: :string, example: 'MyApplication', description: 'Your document upload source identifier as defined in developer portal'
+        property :uuid, type: :string, example: 'b6571888-48b2-436d-b938-9807d8a2a4ef', description: 'A random unique identifier for this submission'
+        property :docType, type: :string, example: 'pension', description: 'Document type, one of "21-22", "526ez"'
+        property :numberPages, type: :number, format: :uint32, description: 'Number of pages in main document'
+        property :hashV, type: :string, description: 'SHA-256 hash of main document contents'
+        property :numberAttachments, type: :number, format: :uint32, description: 'Number of attachments supplied, specify 0 if no attachments'
+        property :numberPages1, type: :number, format: :uint32, description: 'Optional, number of pages in first attachment, if present'
+        property :ahash1, type: :string, description: 'Optional, SHA-256 hash of first attachment contents, if present'
+        property :numberPages2, type: :number, format: :uint32, description: 'Optional, number of pages in second attachment, if present'
+        property :ahash2, type: :string, description: 'Optional, SHA-256 hash of second attachment contents, if present'
+      end
+
+      swagger_schema :DocumentUploadErrors do
+        key :required, [:errors]
+
+        property :errors do
+          key :type, :array
+          items do
+            key :required, %i[title detail code status]
+
+            property :title, type: :string, example: 'Partner Service Error', description: 'Error class'
+            property :detail, type: :string, description: 'Error message', example: 'Duplicate - document already uploaded with that id'
+            property :code, type: :string, enum: %w[
```

Comment:
Going to kill the enum here, since it locks us in with inlining the upload to the central mail api instead of queuing and sending in a background job.